index.html

```diff
@@ -0,0 +1,86 @@
+<link rel="stylesheet" href="styles.css">
+<h1 id="title"> freeCodeCamp Survey Form</h1>
+<p id="description">Thank you for taking the time to help us improve the platform</p>
+
+<form id="survey-form">
+  <div class="input-element">
+    <label id="name-label">
+      Name
+      </label>
+      <br>
+      <input id="name" placeholder="Name"  type="text" required/>
+      </div>
+      <div class="input-element">
+    <label id="email-label">
+      Email
+      </label>
+      <br>
+      <input placeholder="Email" id="email" type="email" required/>
+      </div>
+      <div class="input-element">
+    <label id="number-label">
+      Age(optional)
+      </label>
+ <br>
+      <input placeholder="Age" id="number" min="5" max="100" type="number" required/>
+      </div>
+      <div class="input-element">
+    <label id="select-label">
+      Which option best describes your current role?
+      </label>
+      <br>
+      <select id="dropdown">
+        <option>Student</options>
+        <option>Full time Job</options>
+        <option>Full Time Learner</options>
+        <option>Prefer not say</options>
+        <option>Other</options>
+      </select>
+      </div> 
+      <div>
+        <label>Would you recommend freeCodeCamp to a friend?</label>
+        <div>
+        <label><input type="radio" name="game" value="Definite"/>Definitely</label></div>
+       <div> <label><input type="radio" name="game" value="Maybe"/>Maybe</label></div>
+        <div><label><input type="radio" name="game" value="Not Sure"/>Not sure</label></div>
+      </div>
+      
+    <div class="input-element">
+    <label id="select-label">
+      What is your favorite feature of freeCodeCamp?
+      </label>
+      <br>
+      <select id="dropdown">
+        <option>Challenges</options>
+        <option>Projects</options>
+        <option>Community</options>
+        <option>Open Source</options>
+      
+      </select>
+      </div> 
+      <div>
+        <label>What would you like to see improved? (Check all that apply)</label>
+        <div>
+        <label><input type="checkbox" name="ideas" value="food"/>Additional Courses</label></div>
+       <div> <label><input type="checkbox" name="ideas" value="food"/>Front-end Projects</label></div>
```

Comment:
It is better to add everything in one div instead of open a div tag in every line.

Reply:
- [ ] However much you have stated the changes required, there are few things we need to add in the comment to make our review more professional. I suggest to consider including the `how` and `why` in your comment. This will help the student clearly understand your review comment and apply changes to the exact field requested ☺️.

Reply:
Using div tag to create a section in HTML. Since they are in same part of the page , using one div tag is useful than using in every line.